Chapter5/Chapter5.Task1/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Chapter5.Task1
+{
+    class Program
+    {
+        const int a = 10;
+        private static readonly int num;
+        static Program()
+        {
+            Console.WriteLine("Enter your number");
+            string str = Console.ReadLine();
+            int b = int.Parse(str);
+            num = b;
+        }
+        private void Operation()
+        {
+           int sum = a + num;
+           int dif = a - num;
+           int mult = a * num;
+            Console.WriteLine($"Sum = {sum}, dif = {dif}, mult = {mult} ");
+        }
+        static void Main(string[] args)
+        {
+            Program c = new Program();
```

Comment:
looks strange. It seams for me that you do not understand static constructors.
question: when and how static constructor is called?
advise: make Operation method static and just call it from Main without creation Program instanse.